backend/src/main/java/ca/gov/dtsstn/cdcp/api/service/UserService.java

```diff
@@ -151,7 +151,7 @@ public void updateUser(String userId, User userPatch) {
 
 	public void updateSubscriptionForUser(String userId, String subscriptionId, String languageId) {
 		Assert.hasText(userId, "userId is required; it must not be null or blank");
-		Assert.hasText(subscriptionId, "subscriptionId is required; it must not be null or blank");
+		Assert.hasText(subscriptionId, "subscriptionId is required; it must not be null or blank");		
```

Comment:
```suggestion
		Assert.hasText(subscriptionId, "subscriptionId is required; it must not be null or blank");
```

Reply:
There was one other file that had some trailing whitespace that I didn't bother commenting on because I didn't want to be too nitpicky.

You can configure vscode to automatically remove trailing whitespace by setting this setting: `"files.trimTrailingWhitespace": true`

Reply:
fixed and added the setting